packages/amplify-util-uibuilder/commands/utils/syncAmplifyUiBuilderComponents.js

```diff
@@ -1,24 +1,25 @@
 const aws = require('aws-sdk');
+const { printer } = require('amplify-prompts');
+const { resolveAppId } = require('amplify-provider-awscloudformation');
```

Comment:
this should be called using invokePluginMethod rather than depending on this package directly